Library/Homebrew/test/livecheck/strategy/cpan_spec.rb

```diff
@@ -0,0 +1,21 @@
+# typed: false
+# frozen_string_literal: true
+
+require "livecheck/strategy/cpan"
+
+describe Homebrew::Livecheck::Strategy::Cpan do
+  subject(:cpan) { described_class }
+
+  let(:cpan_url) { "https://cpan.metacpan.org/authors/id/M/MI/MIYAGAWA/Carton-v1.0.34.tar.gz" }
```

Comment:
```suggestion
  let(:cpan_url_no_subdirectory) { "https://cpan.metacpan.org/authors/id/H/HO/HOMEBREW/Brew-v1.2.3.tar.gz" }
  let(:cpan_url_with_subdirectory) { "https://cpan.metacpan.org/authors/id/H/HO/HOMEBREW/brew/brew-v1.2.3.tar.gz" }
```

This brings the test in line with the example URLs in the strategy documentation comment. We also want to make sure that the strategy also applies when the file is found in a subdirectory of the author directory.

Reply:
Yeah, I should've done it. Thank you! 👍